src/analytics/Properties.tsx

```diff
@@ -1485,6 +1486,12 @@ interface TransactionDetailsProperties {
   }
 }
 
+interface WalletJumpstartProperties {
+  [JumpstartEvents.jumpstart_started]: undefined
+  [JumpstartEvents.jumpstart_succeeded]: undefined
+  [JumpstartEvents.jumpstart_failed]: undefined
```

Comment:
yes, `jumpstart_started` is redundant.

added for token/amount tracking separate events:
* `jumpstart_claimed_token` with ERC20 claim details
* `jumpstart_claimed_nft` with ERC721 claim details 

I think it's worth keeping a bare `jumpstart_succeeded` event as well, because getting claim details may fail (but it doesn't mean claim didn't succeeded)